docs/downloads.md

```diff
@@ -153,6 +153,14 @@ The current stable series is **0.12.6**, which was released on June 11, 2020 --
             </td>
             <td colspan="2">&nbsp;</td>
         </tr>
+        <tr>
+            <td><a href="https://aws.amazon.com/amazon-linux-2/">Amazon Linux Lambda</a></td>
+            <td>2</td>
+            <td>
+                <a href="https://github.com/wkhtmltopdf/packaging/releases/download/0.12.6-4/wkhtmltox-0.12.6-4.amazonlinux2_lambda.zip">x86_64</a>
+            </td>
+            <td colspan="4">&nbsp;</td>
+        </tr>
```

Comment:
I'd rather you changed it so that it appears in the last column of AmazonLinux 2 as "lambda zip", so the last `<td colspan="2">` changes to just `<td>&nbsp;</td>` and you have this download there.